test/parallel/test-fs-promises-file-handle-readFile.js

```diff
@@ -35,6 +36,29 @@ async function validateReadFile() {
   await fileHandle.close();
 }
 
+async function validateLargeFileSupport() {
+  const LARGE_FILE_SIZE = 2147483647 + 10 * 1024 * 1024; // INT32_MAX + 10 MB
+  const FILE_PATH = path.join(os.tmpdir(), 'large-virtual-file.bin');
+
+  function createVirtualLargeFile() {
+    return Buffer.alloc(LARGE_FILE_SIZE, 'A');
+  }
+
+  const virtualFile = createVirtualLargeFile();
+
+  await writeFile(FILE_PATH, virtualFile);
```

Comment:
It's also possible that the FS on test runners could completely run out of space with this large of a file

Reply:
I did a check for this place too, thanks for the suggestion